source/jsonnet/england-wales/household/blocks/individual/past_usual_household_address.jsonnet

```diff
@@ -64,7 +64,7 @@ local additionalAnswerOption = [
 
 {
   type: 'Question',
-  id: 'past-usual-household-address',
+  id: 'location-one-year-ago',
```

Comment:
Should the question and answer ids also change?

Reply:
Probably, they didn't match in the first place meaning they were missed.